features/flows/rest/impl/src/main/java/org/opennms/netmgt/flows/rest/internal/FlowRestServiceImpl.java

```diff
@@ -48,6 +48,7 @@
 import javax.ws.rs.core.MultivaluedMap;
 import javax.ws.rs.core.UriInfo;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Can we please not use commons-lang stuff. Try to use google guava or plain java instead.